### PR TITLE
Introduce graph sync multiple

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
Tested in test environment. PTAL

File: thoth/common/openshift.py [hunk 1]
```diff
@@ -1374,6 +1405,52 @@ def schedule_graph_sync_package_extract(
...
+            THOTH_ONLY_DEPENDENCY_MONKEY_DOCUMENTS=int(only_dependency_monkey_documents),
+        )
+        template = self.oc_process(namespace, template)
+        graph_sync = template["objects"][0]
+
+        response = self.ocp_client.resources.get(
+            api_version=graph_sync["apiVersion"], kind=graph_sync["kind"]
+        ).create(body=graph_sync, namespace=namespace)
+
+        _LOGGER.info("Scheduled new graog sync multiple with id %r", response.metadata.name)
```
Comment (harshad16) [inline thoth/common/openshift.py]: there a typo here for graph. :+1:

Comment (fridex) [reply thoth/common/openshift.py]: Nice catch! Should be fixed now.

Comment (harshad16) [reply thoth/common/openshift.py]: I thought it was a test of my review skills :see_no_evil:

**Pull request merged**